web/package.json

```diff
@@ -19,15 +19,17 @@
     "build:clean": "run-s clean compile bundle:dev",
     "build:prod": "run-s clean compile bundle:prod",
     "build:test": "run-s build test",
+    "build-translations": "bux build-translations --src src/explorer-web --dest i18n --outputPath ./dev-server/resources/i18n",
     "bundle": "npm run bundle:dev",
     "bundle:analyze": "webpack --env.analyze",
     "bundle:dev": "webpack --env.dev",
     "bundle:prod": "webpack",
     "bux": "bux",
-    "compile": "tsc -b ./config/tsconfig.build.json",
+    "compile": "run-s build-translations && tsc -b ./config/tsconfig.build.json",
```

Comment:
`run-s` and `&&` are redundant here. I'd switch this to just `npm run build-translations && tsc -b ./config/tsconfig.build.json`